hicommonwealth/commonwealth
# Use existing comment reactions + removed comments/:id/reactions api calls from frontend

## Link to Issue
Closes: #4662

## Description of Changes
The comments RQ (react query) were using the /comments/:id/reactions API to get reactions (we were doing this before comments RQ migration as well).

But this is redundant after #4545 since we removed a lot of unneeded reactions code. 

Now using the comments.reactions[] field on the comment response and updating it when a reaction is added/deleted.

## "How We Fixed It"
Not a bug actually but a refinement.

We refined by using existing data instead of redundant API calls saving bandwidth + improving speed.

## Test Plan
- Add a comment to a reaction, it should update correctly
- Remove comment from a reaction, it should update correctly

## Deployment Plan
N/A

## Other Considerations
This PR should be merged after #4545
